app/controllers/user_address_controller.rb

```diff
@@ -1,8 +1,13 @@
 class UserAddressController < ApplicationController
+  protect_from_forgery except: :index
```

Comment:
Why did you add this protect from forgery?

Reply:
To prevent `CSRF` attack.